applications/jonny5/src/j5_limits.erl

```diff
@@ -64,10 +67,12 @@
 -spec get(ne_binary()) -> limits().
 get(Account) ->
     AccountId = wh_util:format_account_id(Account, 'raw'),
-    case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
-        {'ok', Limits} -> Limits;
+    Limits = case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
+        {'ok', _Limits} -> _Limits;
```

Comment:
After wasting a lot of time I found that my editor infact did save the file
sometimes with CRLF and other times with just LF for line endings.
I found a plugin that will show me what it is using and switch it to what I
want.
So the most recent commit now has both j5_limits.erl and trunks.md with
just LF for line endings. Gotta love netbeans. GRRR. Tried switching to
eclipse a little while back but got hung up on many differences. Maybe I'll
try again.

On Mon, Feb 2, 2015 at 5:03 PM, Dave Singer <daves@1pipetelecom.com> wrote:

> I made a change just to the var name as you suggested (plus copyright and
> contributors) but now for some reason github thinks the whole limits file
> has changed.
> Trying to work that out.
>
> On Mon, Feb 2, 2015 at 11:14 AM, James Aimonetti <notifications@github.com
> > wrote:
>
>> In applications/jonny5/src/j5_limits.erl
>> <https://github.com/2600hz/kazoo/pull/735#discussion_r23949432>:
>>
>> > @@ -64,10 +67,12 @@
>> >  -spec get(ne_binary()) -> limits().
>> >  get(Account) ->
>> >      AccountId = wh_util:format_account_id(Account, 'raw'),
>> > -    case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
>> > -        {'ok', Limits} -> Limits;
>> > +    Limits = case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
>> > +        {'ok', _Limits} -> _Limits;
>>
>> Using the underscore-prefixed variable name is bad practice (other than
>> when logging). Maybe just use {'ok', Lims} -> Lims; or something
>> similarly ghetto :). Using _Limits reads as "This Variable contains the
>> limits but I don't care about it here and it should be ignored", which
>> isn't the case.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/2600hz/kazoo/pull/735/files#r23949432>.
>>
>
>